.env.local

```diff
@@ -0,0 +1,18 @@
+CHAIN_ENV=emulator
+FAUCET_ADDRESS=""
+DATABASE_PATH=./kitty-items-db-emulator.sqlite
+ADMIN_ADDRESS=0xf8d6e0586b0a20c7
+FLOW_PRIVATE_KEY=f8e188e8af0b8b414be59c4a1a15cc666c898fb34d94156e9b51e18bfde754a5
+FLOW_PUBLIC_KEY=6e70492cb4ec2a6013e916114bc8bf6496f3335562f315e18b085c19da659bdfd88979a5904ae8bd9b4fd52a07fc759bad9551c04f289210784e7b08980516d2
+EXT_FLOW_TOKEN_ADDRESS=0x0ae53cb6e3f42a79
+EXT_FUNGIBLE_TOKEN_ADDRESS=0xee82856bf20e2aa6
+EXT_NON_FUNGIBLE_TOKEN_ADDRESS=0xf8d6e0586b0a20c7
+EXT_NFT_STOREFRONT_ADDRESS=0xf8d6e0586b0a20c7
+EXT_METADATA_VIEWS_ADDRESS=0xf8d6e0586b0a20c7
+ACCESS_API=http://localhost:8080
+WALLET_DISCOVERY=http://localhost:8701/fcl/authn
+API_URL=http://localhost:3000
+APP_DOMAIN=http://localhost:3001
+GA_TRACKING_ID=tracking-id
+MIXPANEL_TOKEN=token
```

Comment:
Would it be possible to make these track env vars optional for users, rather than require a placeholder value? i.e. exclude them from .env entirely, but read them in the JS code if they exist?

Reply:
The code will throw an error if these are not present in the `env`. 
I could try to think of a way to make these "optional" without using an `env` flag, but thought this might be the simplest way.